ert3/console/__init__.py

```diff
@@ -66,6 +67,23 @@ def _run_experiment(workspace_root, experiment_name):
     if _experiment_have_run(experiment_root):
         raise ValueError(f"Experiment {experiment_name} have been carried out.")
 
+    coefficients = [
+        (random.gauss(0, 1), random.gauss(0, 1), random.gauss(0, 1))
+        for _ in range(1000)
+    ]
+    data = evaluator(coefficients)
+
+    result = [
+        {
+            "input": {"coefficients": {"a": a, "b": b, "c": c}},
```

Comment:
I think the evaluator should input and output the data structure sketched here instead of the current one. No reason to get rid of all of the context unless we have to :)

Reply:
We argued that the evaluator should not include the entire structure of the data, and rather just return the result. It should be up the caller side to decide on how the input and output data is structured.

We could refactor it out of the `run_experiment` function though

Reply:
After discussion about how the `evaluate` function should behave when we are using it in the ert framework, we've modified the input to also use the same structure. With that said, the complexity of handling said input is now shown to a large degree within the `evaluate` function and the accompanying tests